applications/StructuralMechanicsApplication/python_scripts/structural_response.py

```diff
@@ -409,9 +415,9 @@ def _GetAdjointParameters(self):
 
             # Output process:
             # TODO how to add the output process? How find out about the variables?
-            if adjoint_parameters.Has("output_configuration"):
+            if adjoint_parameters.Has("output_processes"):
                 Logger.PrintInfo("> Output process is removed for adjoint analysis. To enable it define adjoint_parameters yourself.")
-                adjoint_parameters.RemoveValue("output_configuration")
+                adjoint_parameters.RemoveValue("output_processes")
```

Comment:
I think this is quite a weak mechanism, note that not all processes that generate output are in output-processes

Reply:
This automatic setup for the adjoint parameters anyway works only for standard cases. If something more complex comes into play, the adjoint parameters can and should be specified manually.